mod/consensus-types/pkg/types/genesis.go

```diff
@@ -149,21 +154,25 @@ func DefaultGenesisExecutionPayloadHeaderDeneb() (
 	if err != nil {
 		return nil, fmt.Errorf("failed generating receipts root: %w", err)
 	}
+
+	baseFeePerGas, err := math.NewU256FromBigInt(big.NewInt(defaultBaseFeePerGas))
+	if err != nil {
+		return nil, fmt.Errorf("failed setting base fee per gas: %w", err)
+	}
+
 	return &ExecutionPayloadHeader{
-		ParentHash:   common.ExecutionHash{},
-		FeeRecipient: common.ExecutionAddress{},
-		StateRoot:    stateRoot,
-		ReceiptsRoot: receiptsRoot,
-		LogsBloom:    [256]byte{},
-		Random:       common.Bytes32{},
-		Number:       0,
-		//nolint:mnd // default value.
-		GasLimit:  math.U64(30000000),
-		GasUsed:   0,
-		Timestamp: 0,
-		ExtraData: make([]byte, constants.ExtraDataLength),
-		//nolint:mnd // default value.
-		BaseFeePerGas: math.NewU256FromBigInt(big.NewInt(3906250)),
+		ParentHash:    common.ExecutionHash{},
+		FeeRecipient:  common.ExecutionAddress{},
+		StateRoot:     stateRoot,
+		ReceiptsRoot:  receiptsRoot,
+		LogsBloom:     [256]byte{},
+		Random:        common.Bytes32{},
+		Number:        0,
+		GasLimit:      defaultGasLimit,
```

Comment:
nit: moved magic value to a const. Likely outside PR's scope but I rather have all magic values together in one place, rather than spread in the codebase where we use them.